### DISCUSSION TITLE
How can I define a different Probesize value in ffmpeg

### DISCUSSION THREAD

**Discussion by cagnulein** [Motion-Project/motion#1521] [Q&A] [answered]
As the subject, how can I define a different Probesize value in ffmpeg?

**Comment by Mr-Dave** [comment, marked as answer]
Use [netcam_params](https://motion-project.github.io/motion_config.html#netcam_params).  e.g. `netcam_params probesize=whatever`

**Comment by tosiara** [comment]
@Mr-Dave, is it possible to specify `-fflags nobuffer` to the rtsp input the same way?

**Reply by Mr-Dave** [reply]
Yes.  That should work as well.  

Rather than having only a few hard-coded options, Motion now supports specifying any option/value pair in the `netcam_params` and they will be parsed and sent directly into the FFmpeg dictionary.   Therefore if you wanted the `fflags` as well as `probesize` it would be: 

`netcam_params probesize=whatever, fflags=nobuffer`

The Motion log prints out the parsed values so it provides confirmation of how the option/value pair will be sent to FFmpeg.
